optimum/exporters/openvino/model_configs.py

```diff
@@ -147,6 +147,7 @@
     StatefulSeq2SeqDecoderPatcher,
     UpdateCausalMaskModelPatcher,
     XverseModelPatcher,
+    Zamba2ModelPatcher,
 )
 
```

Comment:
would you mind adding a test with a tiny random model ?

Reply:
@rkazants